pkg/kubelet/config/file.go

```diff
@@ -200,3 +201,21 @@ func (s *sourceFile) replaceStore(pods ...*v1.Pod) (err error) {
 	}
 	return s.store.Replace(objs, "")
 }
+
+func checkPlainTextFile(file string) (bool, error) {
+	out, err := exec.Command("file", "--mime-type", file).CombinedOutput()
+	if err != nil {
+		return false, fmt.Errorf("failed to exec file command: %v", err)
+	}
+
+	lines := strings.Split(string(out), ": ")
```

Comment:
I don't think this will work all the time. As you know, in linux, all the characters are valid except `/`